tests/mapping/operation/unmarshal_response_test.py

```diff
@@ -9,8 +9,10 @@ def test_no_content(empty_swagger_spec):
         'description': "I don't have a 'schema' key so I return nothing",
     }
     response = Mock(spec=ResponseLike, status_code=200)
-    status_code, value = unmarshal_response(
-        empty_swagger_spec, response_spec, response)
+    with patch('bravado.mapping.operation.get_response_spec') as m:
+        m.return_value = response_spec
+        op = Mock(swagger_spec=empty_swagger_spec)
```

Comment:
See the purpose of `class Mock(spec=None, ...` @ http://www.voidspace.org.uk/python/mock/mock.html

Two most important reasons this is a good practice:
- Let's future readers know exactly what type of object is being mocked
- Makes sure the unit test doesn't deviate from the type's interface